apis/nucleus/src/stores/create-key-store.js

```diff
@@ -23,6 +25,11 @@ export default (initialState = {}, applyMiddleware = () => {}) => {
     dispatch: (forceNewState) => {
       hookListeners.forEach((listener) => listener(forceNewState ? {} : sharedState));
     },
+    destroy: () => {
+      Object.keys(subscribedListeners).forEach((key) => {
+        subscribedListeners[key]();
+      });
+    },
```

Comment:
I expect this to only work when the applyMiddleware actually returns an unsubscribe function
(it do not for most uses, but for modelStore (only one you call destroy on?) it works)

Reply:
yea, will guard against it